content/english/security-fundamentals/cyber-security-cause-&-effect.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Cyber security: Cause & Effect"
```

Comment:
I don't think the website will like & in the URL - can you rename the file to be `cyber-security-cause-effect`?

Reply:
Renamed it.